uast/semantic-uast.md

```diff
@@ -86,7 +74,21 @@ to access source snippet related to the node.
 
 As an example of a custom positional information, a ternary operator
 `x ? y : z` node may store individual positions for `?` and `:` characters
-as a separate `then` and `else` fields in `Positions` node.
+as a separate `then` and `else` fields in `Positions` node. This field will
+always be in the parent node under the `@pos` property.
```

Comment:
I haven't expected this to be confusing, but from @zurk's feedback it looks like it is. Maybe it's worth to add an example with the serialized `Positions` node? @juanjux What do you think?

Reply:
I was going to add a commented example of semantic UAST to the docs, but in a separate PR.